dist/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Space Burgers</title>
+  </head>
+  <body>
+     <div id="menuid">
```

Comment:
Ideally, the project instructions said to name it `content` which has more semantic meaning given the tabbed content requirement.
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)